### DISCUSSION TITLE
sql transform  sqlalchemy

### DISCUSSION THREAD

**Discussion by zzq120203** [sqlalchemy/sqlalchemy#11320] [Usage Questions]
As the title suggests, I have a puzzle about how this sql is represented：
`SELECT * FROM category2 WHERE (SELECT count(*) FROM json_each(test) WHERE value in (10, 2)) == 2`
I try to do that：
```python
tmp = func.json_each(column).table_valued("value")
select(func.count("*")).where(tmp.c.value.is_(value))
```
didn't get the results I wanted

**Comment by CaselIT** [comment]
Hi,

Not sure what issue you are experiencing. Other than the typo you did in the where (is_ instead of in_) the sql generated is exactly what you provided (for the where subquery that is)
